pkgs/applications/video/qstopmotion/default.nix

```
@@ -2,7 +2,7 @@
 , gettext, pkgconfig, libgphoto2, gphoto2, v4l-utils, libv4l, pcre
 , qwt, extra-cmake-modules }:
 
-mkDerivation rec {
+qt5.mkDerivation rec {
```

please use this formally since you used the callPackage function to bring the qt specific deriver in scope
```suggestion
mkDerivation rec {
```

---

It must have got caught in the rebase. Fixed now.